homeassistant/components/media_player/songpal.py

```diff
@@ -288,7 +288,7 @@ def state(self):
     @property
     def source(self):
         """Return currently active source."""
-        return self._active_source.title
+        return getattr(self._active_source, 'title', None)
```

Comment:
Could you add a single-line comment here describing the necessity of using getattr instead of regular attribute access here, then it's okay to merge.

Reply:
Sure - here you go. Is this comment sufficient?

Reply:
That's good, thanks!